src/SubsMgr.cpp

```diff
@@ -684,6 +687,19 @@ SubStatus TransactionSubsMgr::getFullStatus(const HashX &txHash) const
     return storage->getTxHeight(txHash);
 }
 
+ScriptHashTransactionsSubsMgr::~ScriptHashTransactionsSubsMgr() {}
+
+auto ScriptHashTransactionsSubsMgr::getFullStatus(const HashX &sh) const -> SubStatus
+{
+    auto [mempool, lock] = storage->mempool();
+    const auto shIt = mempool.scriptHashTransactionsAffected.find(sh);
+    if (shIt == mempool.scriptHashTransactionsAffected.end()) {
+        return {};
```

Comment:
What will happen upon empty response? The notification will not depart to subscriber?